client/widgets/AdventureMapClasses.cpp

```diff
@@ -1062,7 +1062,7 @@ void CInGameConsole::keyPressed (const SDL_KeyboardEvent & key)
 				captureAllKeys = false;
 				endEnteringText(false);
 			}
-			else if(SDLK_TAB)
+			else if(SDLK_TAB == key.keysym.sym)
```

Comment:
Unlike all others this one could actually be meaningful fix. E.g we have leak of "S" hotkey when typing cheats fast and this might be it. Hopefully it's source of the problem.